client/src/App.js

```diff
@@ -1,19 +1,31 @@
-import React from 'react';
+import React, {useState} from 'react';
 import useApplicationData from './hooks/useApplicationData';
 import './App.css';
+import LeftSidebar from './components/LeftSidebar/LeftSidebar';
```

Comment:
I guess it is better to import these components after we create them and want to use them. In this branch the purpose is not to import or create these components.